apps/dsp-app/cypress/e2e/system-admin/data-model.cy.ts

```diff
@@ -0,0 +1,61 @@
+import { faker } from '@faker-js/faker';
+import ProjectPage from '../../support/pages/project-page';
+
+describe('Data Model', () => {
```

Comment:
Shouldn't we call this ontologies ?

Reply:
Yeah.. its a bit messed up all over the project. routes, api and some labels called - data model, some other labels called ontology. Are you sure I need to rename it?

Reply:
I think ontologies seems better, it's everywehere in the app and everybody talks about ontology